syntax-test_dev.py

```diff
@@ -0,0 +1,271 @@
+import sublime
+import sublime_plugin
+from os import path
+import re
+
+
+def get_syntax_test_tokens(view):
```

Comment:
Since all "helper" functions revolve around a given view,
it could make sense architecture-wise to group the functions within a class and store instances of these in a way that the plugin interface points (commands and event listeners) can access it.

Just a thought. I don't know if this would end up being better.